templates/common/_base/units/machine-config-daemon-pull.service.yaml

```diff
@@ -18,8 +18,8 @@ contents: |
   # See https://github.com/coreos/fedora-coreos-tracker/issues/354
   ExecStart=/bin/sh -c '/bin/mkdir -p /run/bin && chcon --reference=/usr/bin /run/bin'
   ExecStart=/bin/sh -c "while ! /usr/bin/podman pull --authfile=/var/lib/kubelet/config.json --quiet '{{ .Images.machineConfigOperator }}'; do sleep 1; done"
-  ExecStart=/bin/sh -c "/usr/bin/podman run --rm --quiet --net=host --entrypoint=cat '{{ .Images.machineConfigOperator }}' /usr/bin/machine-config-daemon > /run/bin/machine-config-daemon.tmp"
-  ExecStart=/bin/sh -c '/usr/bin/chmod a+x /run/bin/machine-config-daemon.tmp && mv /run/bin/machine-config-daemon.tmp /run/bin/machine-config-daemon'
```

Comment:
The previous code was making some effort to do this atomically.  If interrupted, `cp` will leave a half-written file.

Perhaps we should instead add an `install-to-host` verb and implement all this in Go and not shell script.

Dunno.  I guess for now atomicity of this doesn't matter because we aren't trying to be idempotent, so

/approve

Reply:
Right, systemd would restart the service and stomp out the incomplete binary so I'm not too worried about this